src/GPU3D.h

```diff
@@ -197,7 +197,7 @@ class GPU3D
 
     FIFO<CmdFIFOEntry, 64> CmdStallQueue {};
 
-    u32 ZeroDotWLimit = 0;
+    u32 ZeroDotWLimit = 0xFFFFFF;
```

Comment:
doesn't this need to be changed in Reset?

Reply:
i did change it there, i just now also changed the variable's default value to look pretty/avoid confusion.

Reply:
though looking at it now no other variable gets initialized in gpu3d.h to its actual default as stated in the reset function
so maybe i should revert that change to avoid inconsistency?

Reply:
there isn't really a reason for the variables to be initialised in the header, except to prevent UB in case they are read before Reset is called (though that shouldn't happen if that makes any sense).

Reply:
yeah, so it's mostly just personal preference what value they get initialized to
i wont worry about it
if someone wants it changed back for consistency with the other values being 0 initialized I can revert that change easily